Asch_Harwood Project_Proposal

AschHarwood opened the pull request.

AschHarwood committed:
created a test file..need to save to posts file

AschHarwood committed:
copied hw1 and renamed template

AschHarwood committed:
finished hw1

AschHarwood committed:
project proposal

SidharthSaluja reviewed (commented):
Umm csv file should be in the posts/_data folder. It should not be directly in the posts folder

AschHarwood committed:
moved data folder

AschHarwood committed:
updated data link to github url

AschHarwood committed:
removed hw2 file

AschHarwood commented:
HI @SidharthSaluja 

I believe I fixed it. Also, i realized I committed a draft of hw 2 that I was ready to submit. I believe I also removed that too.
